src/js/components/Video/Video.js

```diff
@@ -161,9 +161,10 @@ const Video = forwardRef(
         const { textTracks } = video;
         if (textTracks.length > 0) {
           if (textTracks.length === 1) {
-            const active = textTracks[0].mode === 'showing';
+            const track = textTracks[0];
+            const active = track.mode === 'showing';
             if (!captions || !captions[0] || captions[0].active !== active) {
-              setCaptions([{ active }]);
+              setCaptions([{ label: track.label, active }]);
```

Comment:
maybe add a comment as to even if there is one `track` we still want to grab that  `label` if there  is one

Reply:
Added a comment here as well as adding comments a few other places in the code.